js/renovation/ui/scroll_view/scroll_view_props.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  ComponentBindings,
+  OneWay,
+} from 'devextreme-generator/component_declaration/common';
+
+import {
+  ScrollableProps,
+} from './scrollable_props';
+
+import BaseWidgetProps from '../../utils/base_props';
+import messageLocalization from '../../../localization/message';
+
+@ComponentBindings()
+export class ScrollViewProps extends ScrollableProps {
+  @OneWay() reachBottomText: string = messageLocalization.format('dxScrollView-reachBottomText');
```

Comment:
It is not used, reachBottomText is not rewritten

Reply:
Fixed in https://github.com/DevExpress/DevExtreme/pull/15914/commits/29a91afaca19b72b35c608c1b3171332a569590d